juniper_hyper/src/lib.rs

```diff
@@ -357,44 +430,58 @@ mod tests {
 
     #[test]
     fn test_hyper_integration() {
```

Comment:
I believe because you use the `macros` feature you can use `#[tokio::test]` on an async fn.